docs/documentation/index.html

```diff
@@ -81,79 +81,118 @@
+
+
+					<div class="row" style="height:80px;"></div>
+
+					<h1>Getting Started</h1>
+
+					<p>This short introduction will briefly guide you through the project setup and the user interface of Sourcetrail. The bullet point lists will provide you with the next steps to take:</p>
+					
+					<!-- <div class="callout callout-info">
+						<strong>Tasks:</strong>
+						<ul>
+							<li>Continue reading.</li>
+						</ul>
+					</div> -->
+
+					<h2>Starting Up Sourcetrail</h2>
+					
+					<p>Once you've <a href="https://github.com/CoatiSoftware/Sourcetrail/releases">downloaded</a> Sourcetrail successfully you are ready to run the application. For assistance wth installation, visit the <a href="#Installation">installation</a> section.</p>
 
-					<h2>Starting up Sourcetrail</h2>
-					<p>Once you <a href="#INSTALLATION">installed</a> Sourcetrail successfully you are ready to run the application.</p>
 					<div class="callout callout-info">
 						<strong>Tasks:</strong>
 						<ul>
-							<li>Run Sourcetrail</li>
+							<li>Launch Sourcetrail.</li>
+						</ul>
+					</div>
+
+					<!-- <h4>Start Window</h4> -->
+
+					<p>After launching Sourcetrail you will see the <a href="#StartWindow">Start Window</a>. From here you can create your own project or choose a pre-indexed one.</p>
+
+						<div class="callout callout-info">
+						<strong>Tasks:</strong>
+						<ul>
+							<li>Click <b>New Project</b> to create a new project.</li>
+							<li>or select one from the <b> Recent Projects </b> <i> (ex: TicTacToe) </i> and continue with the <a href="#UIIntro">UI Intro</a>.</li>
 						</ul>
 					</div>
 
-					<h4>Start Window</h4>
-					<p>After launching Sourcetrail you will see the <a href="#StartWindow">Start Window</a>. Now you can create your own project or choose a pre-indexed one.</p>
 					<div class="row">
 						<div class="col-sm-8 col-sm-offset-2">
 							<img src="img/start_window.png" style="width:100%;">
 						</div>
 					</div>
+
+
+					<h2>Creating a New Project</h2>
+					
+					<p>When creating a new Sourcetrail project you will use the <a href="#ProjectSetupWizard"> Project Setup Wizard </a>. This wizard splits the setup process into several subsequent steps. Depending on your project's structure and the used build system, there are different types for project setup. Choosing the correct setup method is important and can make the setup process a lot easier.</p>
+
+					<!-- <p>In the first step of the project setup, you will just need to specify name and location of your Sourcetrail project.</p> -->
```

Comment:
You are right. Giving this information only in the tasks list box is probably enough. So you can also remove this.